packages/jwt-verifier/test/integration-test/resource-server.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const OktaJwtVerifier = require('../../lib');
+var cors = require('cors');
+
+const oktaJwtVerifier = new OktaJwtVerifier({
+  issuer: 'http://localhost:9090/oauth2/default',
+  assertClaims: {
+    aud: 'api://default'
+  },
+});
+
+/**
+ * A simple middleware that asserts valid access tokens and sends 401 responses
+ * if the token is not present or fails validation.  If the token is valid its
+ * contents are attached to req.jwt
+ */
+function authenticationRequired(req, res, next) {
+  const authHeader = req.headers.authorization || '';
+  const match = authHeader.match(/Bearer (.+)/);
+
+  if (!match) {
+    res.set('WWW-Authenticate', 'Bearer realm=resource error="unauthorized", error_description="Bearer token not found');
+    return res.status(401).end();
+  }
+
+  const accessToken = match[1];
+
+  return oktaJwtVerifier.verifyAccessToken(accessToken)
+    .then((jwt) => {
+      req.jwt = jwt;
+      next();
+    })
+    .catch((err) => {
+      const header = 'Bearer realm=resource, error="unauthorized", error_description="' + err.message + '"';
```

Comment:
nit: use template string for templates when possible:

```typescript
const header = `Bearer realm=resource, error="unauthorized", error_description="${err.message}"`;
```

Reply:
Done